libraries/achartengine/achartengine/src/org/achartengine/GraphicalActivity.java

```diff
@@ -1,12 +1,12 @@
 /**
  * Copyright (C) 2009 - 2013 SC 4ViewSoft SRL
- *  
+ * <p/>
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *  
- *      http://www.apache.org/licenses/LICENSE-2.0
- *  
+ * <p/>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p/>

```

Comment:
Why are these html tags here?
In my opinion, javadocs should not have html tags. We read them at the source level, and pretty much never compile them to html, so adding hard to read html tags is not useful.